<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>610</width>
      <height>170</height>
     </rect>
    </property>
    <property name="title">
     <string>Genetic Algorithm Configuration</string>
    </property>
    <widget class="QLineEdit" name="edPopSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>76</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Population Size:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edDNASize">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DNA Size:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edCrossProb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edMutProb">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edElitimsRatio">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edCrossScale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edMutScale">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edElitimsScale">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCromType">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Byte</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Signed Int</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unsigned Int</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Float</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Double</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbCrossType">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>One Cut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Two Cut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cut Splice</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btStart">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>90</y>
       <width>140</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btStop">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>140</y>
       <width>140</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>93</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Chromossome Size:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>79</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Crossover Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>106</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Crossover Probability:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Crossover Scale Factor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mutation Scale Factor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Elitism Scale Factor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>99</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mutation Probability:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Elitism Ratio:</string>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="cpSpace" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>611</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
